code/game/objects/random/maintenance.dm

```diff
@@ -106,7 +106,8 @@ something, make sure it's not in one of the other lists.*/
 				prob(1);/obj/item/weapon/card/emag_broken,
 				prob(2);/obj/item/device/camera,
 				prob(3);/obj/item/device/pda,
-				prob(3);/obj/item/device/radio/headset)
+				prob(3);/obj/item/device/radio/headset,
+				prob(5);/obj/item/pizzavoucher)
```

Comment:
I would drop this down to prob(1), gotta have it be **memorable** when someone finds it in maint.